src/axom/quest/InOutOctree.hpp

```diff
@@ -1540,6 +1572,9 @@ void InOutOctree<DIM>::insertMeshTriangles()
           childBlk[j] = blk.child(j);
           childBB[j] = this->blockBoundingBox(childBlk[j]);
 
+          // expand bounding box slightly to deal with grazing triangles
+          childBB[j].scale(m_boundingBoxScaleFactor);
```

Comment:
Note: This line is the bugfix. It inflates octree block bounding boxes by a small factor (e.g. 1.000123) before testing for intersections with the mesh triangles. This prevents these triangles from "slipping through the cracks" :)